.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 120
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: >
+  Closed by probot-stale due to  a lack of recent activity. Please feel free
```

Comment:
Not sure if anyone cares, but there's an extra space between "to" and "a" on this line.

Reply:
D'oh! Let's see how that looks once the bot starts commenting on issues. HTML generally doesn't care about whitespace.